distribution/lib/Standard/Table/0.0.0-dev/src/Column.enso

```diff
@@ -2971,3 +2979,13 @@ apply_unary_map column:Column new_name:Text function expected_result_type:Value_
 Many_Files_List.from (that : Column) =
     Read_Many_Helpers.ensure_column_type_valid_to_be_files_list that <|
         Many_Files_List.Value that that.to_vector
+
+## PRIVATE
+   Temporary function to determine if to use the Comparators statics or embedded operations.
+call_comparator left other new_name java_static enso_action =
+    use_java = case left.value_type of
+        Value_Type.Date -> True
+        Value_Type.Time_Of_Day -> True
+        _ -> False
```

Comment:
What is the meaning of this logic?

Reply:
These are the migrated value_types.

Reply:
ah ok it determines which comparator is supported

Reply:
Maybe we could do this in Java, something like `Comparators.EQ.isSupported(column)`?

Reply:
Right now a similar check is done inside of the comparator, just on the Java level. So we need to put in extra effort to ensure that the two places stay in sync.

I think it would be better to directly defer to Java so that 'what is supported' is defined in a single place.

Reply:
It will all be in the Java static in 2 PRs.

Reply:
Put into Java